### DISCUSSION TITLE
New Source: SAP Hana

### DISCUSSION THREAD

**Discussion by lucas-wrp** [airbytehq/airbyte#33162] [New Connector Request]
### Connector Name

SAP HANA 

### What type of integration

Source

### Contribute

- [X] Yes, I want to contribute

**Comment by jrosskopf** [comment]
If you guys are not necessarily interested in SAP HANA via the database, but higher level integration on **SAP ERP tables**, **BAPIs**, but also **SAP Business Warehouse (BW)** cubes and querys and **ODP data sources** (e.g. CDS views, but also HANA tables), then maybe have a look at: https://github.com/datazoode/erpl-airbyte-connectors

We just did an experiment with Airbyte to see whether there is demand from the community. In case you are interested, open an issue in the repo or directly contact us via: https://erpl.io/contact.html

[a comment by an automated account is not shown]
